examples/02_Parameter Estimation.ipynb

```diff
@@ -13,38 +13,54 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Parameter estimation is used to tune the parameters of a general model so its behavior matches the behavior of a specific system. Parameter Estimation can be considered more of an art than a science. For example, parameters of the battery model can be tuned to configure the model to describe the behavior of a specific battery.\n",
+    "Parameter estimation is used to tune the parameters of a general model so that its behavior matches that of a specific system. Parameter Estimation can be considered more of an art than a science. For example, the parameters of a battery model can be tuned to configure the model to more accurately describe the behavior of a specific battery.\n",
```

Comment:
```suggestion
    "Parameter estimation is used to tune the parameters of a general model so that its behavior matches that of a specific system. For example, the parameters of a battery model can be tuned to configure the model to more accurately describe the behavior of a specific battery.\n",
```
That sentence felt a little awkward here